torchvision/prototype/transforms/functional/_color.py

```diff
@@ -217,14 +216,16 @@ def _rgb_to_hsv(image: torch.Tensor) -> torch.Tensor:
 
 def _hsv_to_rgb(img: torch.Tensor) -> torch.Tensor:
     h, s, v = img.unbind(dim=-3)
-    h6 = h * 6
+    h6 = h.mul(6)
     i = torch.floor(h6)
-    f = h6 - i
+    f = h6.sub_(i)
```

Comment:
Minor optimizations based on @pmeier's finding that mul is preferable to `*` for numbers. Also `h6` can be modified in-place as it's not reused.